lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -597,7 +597,13 @@ def generate_clone_spec(src_config)
 
     if get_config(:customization_vlan)
       vlan_list = get_config(:customization_vlan).split(',')
-      networks = vlan_list.map { |vlan| find_network(vlan) }
+      networks =
+        if get_config(:customization_sw_uuid)
+          sw_uuid = get_config(:customization_sw_uuid)
```

Comment:
My experience with using auto for the dvswitch didn't work out well, so I'm not sure exactly how it's supposed to be used, so I think maybe ignoring it is the best thing for now?

I'm also interested in the annotation idea but our VMs are only getting attached to a single VLAN at a time, so I don't really have a great test case... Not sure my design would necessarily reflect how people use that setting...